support-metrics-common/src/test/java/io/confluent/support/metrics/submitters/KafkaSubmitterTest.java

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.confluent.support.metrics.submitters;
+
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import io.confluent.support.metrics.common.kafka.ZkUtilsProvider;
+import kafka.utils.ZkUtils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+
+
+public class KafkaSubmitterTest {
+
+  private static ZkUtilsProvider zkUtilsProvider;
```

Comment:
This could be non-static and the below method/annotation `@BeforeMethod`. No reason for it to be `@BeforeClass` as it isn't starting a server or anything